typescript/vrotsc/src/compiler/transformer/fileTransformers/workflow/decorators/decisionItemDecoratorStrategy.ts

```diff
@@ -57,7 +64,49 @@ export default class DecisionItemDecoratorStrategy implements CanvasItemDecorato
 		return this.sourceFilePrinter.printSourceFile(methodNode, sourceFile);
 	}
 
+	/**
+	 * Prints out the decision item
+	 *
+	 * - `out-name` is the target if the condition is met
+	 * - `alt-out-name` is the target if the `else` condition is met
+	 *
+	 * The `sourceText` is cleared of the `wrapper` function, if it exists
+	 *  it was only added to the sourceText to make the code valid for typescript
+	 *
+	 * @param pos The position of the item in the workflow
+	 * @returns The string representation of the decision item
+	 */
 	printItem(pos: number): string {
-		throw new Error("Method not implemented.");
+		const stringBuilder = new StringBuilderClass("", "");
+		const targetItem = findTargetItem(this.itemInfo.target, pos, this.itemInfo);
```

Comment:
I think it is wise to check whether the `targetItem` is null

Reply:
Done